aviatrix_ha.py

```diff
@@ -394,6 +416,7 @@ def set_environ(client, lambda_client, controller_instanceobj, context,
         # Tags starting with aws: is reserved
         if not key.startswith("aws:"):
             tags_stripped.append(tag)
+    target_group_arns = get_target_group_arns(inst_id)
```

Comment:
set_environ should be just update the lambda env variables.
You could call  target_group_arns = get_target_group_arns(inst_id) outside set_environ and update os.environ['TARGET_GROUP_ARNS'] and then call set_environ in the regular code flow

Reply:
Removed get_target_group_arns() from this function.